README.md

```diff
@@ -151,7 +159,7 @@ Feel free to check the [issues page](../../issues/).
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Without Microverse Help this project not compleated at all so thanks a lot Microverse for giving us this kind of opertunity.
+> Specials thanks Nelson Sakwa who came up with this amazing original design
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
### README LICENCE 

- [ ] This issue is not yet resolved :heavy_multiplication_x: 

   - As stated by the previous reviewer, please, kindly add a licence to your readme to make it more professional as the link is currently broken.

:point_down: 
*previous reviewer's required changes*
![Screenshot from 2023-02-10 15-27-42](https://user-images.githubusercontent.com/59397159/218116666-eb77fc13-297e-40cf-b6ae-8acc5386b278.png)

Reply:
Thanks a lot @zieeco